hack/tools/log-push/main.go

```diff
@@ -329,6 +329,18 @@ func prepareLogsForLoki(ld LogData, logJSONAdditionalLabels []string) ([]LokiStr
 		// We intentionally silently ignore the error, otherwise we
 		// would get too many errors with logs in text format.
 		if err == nil {
+			// Store the ts in original_ts so that it's shown as a separate k/v in Loki.
+			if parsedLogLine.Exists("ts") {
+				originalTimestampMillis := parsedLogLine.Get("ts")
+				parsedLogLine.Set("original_tsMs", originalTimestampMillis)
+
+				t := time.UnixMilli(int64(originalTimestampMillis.GetFloat64()))
+				originalTimestamp := strconv.Quote(t.Format(time.RFC3339Nano))
+				parsedLogLine.Set("original_ts", fastjson.MustParse(originalTimestamp))
+
+				lokiStream.Values[0][1] = parsedLogLine.String()
```

Comment:
Q given my limited understanding of loki stream: Are we sure at the stage that element at [0][1] exist? (no index out of bounds)

Reply:
Currently yes, as we just set it above.

But it's a brittle assumption to make going forward. I actually thought of this, but just forgot to fix it.

I'm now just modifying logline and setting it below